functions/packages.bash

```diff
@@ -572,7 +572,7 @@ nginx_setup() {
   if cond_redirect sed -i -e 's|DOMAINNAME|'"${domain}"'|g' /etc/nginx/sites-enabled/openhab; then echo "OK"; else echo "FAILED (set domain name)"; return 1; fi
 
   if [[ $auth == "true" ]]; then
-    if openhab2_is_installed; then
+    if openhab_is_installed; then
```

Comment:
This is actually not needed, if you look closely this is actually just provided for backwards  compatibility with OH2 the routine already accounts for OH3 on line 582.

Reply:
You are right I have overseen this line. I just understood that for OH3 another method (`Header Set-Cookie`) is used. The nginx config does not use the credentials anyway (only in OH2 case). See https://github.com/openhab/openhabian/blob/a1800e5583e2cc7edb298da8d175f070168ac5bf/functions/packages.bash#L575-L579
But then it does not makes sense to show the auth prompt anyway for OH3? At this point my little change reveals a lot of questions to me and I'm not sure what the intended results for OH2 and OH3 cases are. Now that you are aware of the issue someone may reconsider the setup for this step.

Reply:
OK, I see what's happened here, when we were making the upgrade to OH3 we made a mistake in the coding for this function. I will go ahead and fix it is a commit and incorporate this change as well if you don't have any issue with it.